src/main/api/2/studio-api-2.yaml

```diff
@@ -3287,6 +3412,320 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /dashboard/data:
```

Comment:
Break it out to many dashboard APIs:
- Content
- Publishing
- Audit
- Workflow
- User
- Group

Design first and we can implement what we can over time.